gammapy/modeling/models/core.py

```diff
@@ -484,6 +503,9 @@ def write(
             Overwrite templates FITS files. Default is False.
         write_covariance : bool, optional
             Whether to save the covariance. Default is True.
+        checksum : bool
```

Comment:
Is it really worth having this options?  It think we should be strict on write and always add a checksum. I can see not harm in adding it...

Reply:
I agree

Reply:
Again I have followed the astropy handling here. And now all gammpy methods writing FITS files have a boolean `checksum` argument. See #4837